Doc/whatsnew/3.10.rst

```diff
@@ -554,8 +554,9 @@ the class name followed by an argument list resembling a constructor. This
 pattern has the ability to capture class attributes into variables::
 
     class Point:
-        x: int
-        y: int
+        def __init__(self, x, y):
+            self.x = x
+            self.y = y
```

Comment:
I'm not sure about this, but the description above is about "class attributes" and not "instance attributes". I didn't check whether the pattern is legit or not but maybe check that the current code actually works?